modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/aggregation/ElasticsearchAggregationVisitor.java

```diff
@@ -90,7 +90,7 @@
  */
 @Component(
 	property = "search.engine.impl=Elasticsearch",
-	service = {AggregationTranslator.class, AggregationVisitor.class}
+	service = AggregationTranslator.class
 )
 public class ElasticsearchAggregationVisitor
```

Comment:
@MrJohn1911 Now these classes are providing service for `*Translator`, so their names should be `*Translator` instead of `*Visitor`.